src/js/vre/chapter36/config/form.js

```diff
@@ -32,60 +32,46 @@ const formConfig = {
   chapters: {
     applicantInformation: {
       title: 'Applicant Information',
+      initialData: {
+        socialSecurityNumber: '424242424'
+      },
       pages: {
         applicantInformation: {
           title: 'Applicant information',
           path: 'applicant-information',
           uiSchema: {
-            'view:isVeteran': {
-              'ui:title': 'Are you a Servicemember or Veteran applying for counseling service?',
+            socialSecurityNumber: ssnUI,
+            gender: {
+              'ui:title': 'Gender',
+              'ui:widget': 'radio'
+            },
+            seekingRestorativeTraining: {
+              'ui:title': 'Are you a child who is at least 14 years old, a spouse, or a surviving spouse with a disability and looking for special restorative training?',
+              'ui:widget': 'yesNo'
+            },
+            seekingVocationalTraining: {
+              'ui:title': 'Are you a child, a spouse, or a surviving spouse with a disability and looking for special vocational training',
               'ui:widget': 'yesNo'
             },
-            applicantFullName: _.merge(fullNameUI, {
-              first: {
-                'ui:required': formData => formData['view:isVeteran'] === false,
-              },
-              last: {
-                'ui:required': formData => formData['view:isVeteran'] === false,
-              },
-              'ui:options': {
-                expandUnder: 'view:isVeteran',
-                expandUnderCondition: false
-              }
-            }),
-            applicantRelationshipToVeteran: {
-              'ui:title': 'What is your relationship to the Servicemember or Veteran?',
-              'ui:widget': 'radio',
-              'ui:required': formData => formData['view:isVeteran'] === false,
-              'ui:options': {
-                expandUnder: 'view:isVeteran',
-                expandUnderCondition: false
-              }
+            receivedPamphlet: {
+              'ui:title': 'Have you received a pamphlet explaining survivors\' and dependents\' educational assistance benefits?',
+              'ui:widget': 'yesNo'
             }
           },
           schema: {
             type: 'object',
-            required: ['view:isVeteran'],
+            definitions: {
+              seekingRestorativeTraining,
+              seekingVocationalTraining,
+              receivedPamphlet
+            },
             properties: {
-              'view:isVeteran': {
-                type: 'boolean'
-              },
-              applicantFullName: _.unset('required', applicantFullName),
-              applicantRelationshipToVeteran: {
-                type: 'string',
-                'enum': [
-                  'Spouse',
-                  'Surviving spouse',
-                  'Child',
-                  'Stepchild',
-                  'Adopted child'
-                ]
-              }
+              socialSecurityNumber: ssn,
+              gender
```

Comment:
The hca form had a more complicated property for gender: https://github.com/department-of-veterans-affairs/vets-website/blob/a5bb293bbc8fe6fb4fa04938e5cf4a2f2892c3b6/src/js/hca/config/form.js#L272-L276

My understanding is that this is mapping the keys between the form UI and fullSchema36 and doing some other validation related stuff. What's different about the hca form's approach and just using the definition from the full schema?

Reply:
I figured this out. `gender` in the `fullSchema36` are not user friendly strings.